l10n/en/products/vpn/shared.ftl

```diff
@@ -0,0 +1,75 @@
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/vpn/
+
+vpn-shared-product-name = { -brand-name-mozilla-vpn }
+vpn-shared-subscribe-link = Get { -brand-name-mozilla-vpn }
+vpn-shared-waitlist-link = Join the Waitlist
+vpn-shared-sign-in-link = Already a subscriber?
+vpn-shared-available-countries = We currently offer { -brand-name-mozilla-vpn } in the US, the UK, Canada, New Zealand, Singapore and Malaysia.
+vpn-shared-available-countries-v2 = We currently offer { -brand-name-mozilla-vpn } in the US, the UK, Germany, France, Canada, Malaysia, New Zealand, and Singapore.
+vpn-shared-availablity-long = The VPN is currently available for { -brand-name-windows } 10, { -brand-name-android } and { -brand-name-ios } devices in the US, the UK, Canada, New Zealand, Singapore and Malaysia.
+vpn-shared-money-back-guarantee = 30-day money-back guarantee
+vpn-shared-featured-in = Featured in
+
+vpn-shared-features-encrypt = Device-level encryption
+
+# Variables:
+#   $servers (number) - number of available servers
+#   $countries (number) - number of available countries
+vpn-shared-features-servers = { $servers }+ servers in { $countries }+ countries
+
+vpn-shared-features-bandwidth = No bandwidth restrictions
+vpn-shared-features-activity = No logging of your network activity
+
+# Variables:
+#   $devices (number) - maximum number of connected devices
+vpn-shared-features-devices = Option to connect up to { $devices } devices
+
+# Variables:
+#   $servers (number) - number of available servers
+#   $countries (number) - number of available countries
+vpn-shared-features-server-countries = Connect to more than { $servers } servers in over { $countries } countries
+
+# Variables:
+#   $countries (number) - number of available countries
+vpn-shared-countries-coming-soon = Available in { $countries } countries now. More regions coming soon
+
+# Variables:
+#   $url (number) - link to https://mullvad.net/servers/
+#   $attrs (string) - specific attributes added to external links
+vpn-shared-features-full-list-servers = See our full list of <a href="{ $url }" { $attrs }>servers</a>.
+
+vpn-shared-refund-policy = Refund Policy
+vpn-shared-privacy-notice = Privacy Notice
+vpn-shared-terms-conditions = Terms and Conditions
+vpn-shared-wireguard-copyright = { -brand-name-wireguard } is a registered trademark of Jason A. Donenfeld
+
+## Pricing options. Some offers may only shown in select countries (e.g. German and France).
+
+vpn-shared-pricing-variable-heading = Choose a plan that works for you
+vpn-shared-pricing-variable-sub-heading = All of our plans include:
+vpn-shared-pricing-recommended-offer = Recommended
+vpn-shared-pricing-plan-6-month = 6 Month
+vpn-shared-pricing-plan-12-month = 12 Month
+vpn-shared-pricing-plan-monthly = Monthly
+
+# Variables:
+#   $amount (string) - monthly subscription price using localized currency e.g. 'US$4.99/month'
```

Comment:
I assume `$amount (string)` is considered a string instead of a number is because of the currency associated with the number? And the currency can be automatically generated, depending the currency.

Reply:
Yes we'll provide both the amount and the currency. I'll make this clearer in the comment